qcelemental/models/molecule.py

```diff
@@ -333,11 +333,35 @@ def pretty_print(self):
 
         return text
 
-    def get_fragment(self, real, ghost=None, orient=False):
-        """
-        A list of real and ghost fragments:
-        """
+    def get_fragment(self,
+                     real: Union[int, List],
+                     ghost: Optional[Union[int, List]] = None,
+                     orient: bool = False,
+                     group_fragments: bool = True) -> 'Molecule':
```

Comment:
What do you think about changing the default behavior of this to False after we update Fractal?

Reply:
Yes, I'm all for that so long as Fractal stays happy. The current default is, to me, unexpected and slippery for gradients.